doc/database/hierarchy.md

```diff
@@ -92,14 +92,42 @@ Either two parent binomials, separated by a "x" or "×" or a given binomial, wit
 (see [Wikipedia](https://en.wikipedia.org/wiki/Hybrid_name)).
 So the name does not necessarily say if a plant is a hybrid.
 
-### Rendering
+### Rendering the Unique Name
 
-The unique name must be rendered:
+The unique name of a plant needs to be displayed in the following manner:
 
-- Latin name in _italics_.
-- Cultivar name normal font in single quotes (as in database).
+- Latin name should be displayed in _italics_.
+- The cultivar name should be in regular font, enclosed in single quotes (as stored in the database).
 
-E.g.: _Brassica oleracea italica_ 'Ramoso calabrese'
+For example: _Brassica oleracea italica_ 'Ramoso calabrese'
+
+### Rendering the Complete Name
+
+The complete name of a plant should be displayed as follows:
+
+- If there is a common name, the common name appears first and should start with a capital letter (e.g., "tomato" becomes "Tomato").
+- If it is a plant name from the seed database, the name continues with hyphen `-` and the additional name.
+- This is followed by the unique name in brackets, unique name as described above in `Rendering the Unique Name`.
+
+The general format is:
+
+Common name - additional name (_unique name_ 'cultivar name')
+
+In cases where the common name is not available in the current language, it should be displayed as:
+
+_unique name_ 'cultivar name' - additional name
+
+The cultivar is a part of the unique name.
+
+For example:
+
+- Italian broccoli - violett (_Brassica oleracea italica_ 'Ramoso calabrese')
+- Italian broccoli - violett (_Brassica oleracea italica_)
+- Italian broccoli (_Brassica oleracea italica_)
+- _Brassica oleracea italica_ 'Ramoso calabrese' - violett
+- _Brassica oleracea italica_ 'Ramoso calabrese'
+- _Brassica oleracea italica_ - violett
+- _Brassica oleracea italica_
 
```

Comment:
@badnames does this help?

```suggestion

Usage of complete name:

- The common name alone is enough for overview, e.g. when you see all labels in the map editor at once.
- The complete name should be always be given if the user wants to know precisely which plant, e.g. in search results or in plant details.
- The additional name needs to be additionally given if a plant is connected with a seed.

```

Reply:
Added that and extended it a bit

Reply:
Great, looking forward to the other changes I already proposed earlier to get this merged&done.

Reply:
should all be implemented now